server/clip_server/executors/clip_onnx.py

```diff
@@ -25,6 +26,7 @@ def __init__(
         minibatch_size: int = 32,
         access_paths: str = '@r',
         model_path: Optional[str] = None,
+        drop_image_content: bool = False,
```

Comment:
`drop_image_content` should not be as init argument. Let's take it as a request parameter only.

Reply:
ok make sense